src/audiounit.cs

```diff
@@ -273,7 +304,20 @@ interface AUAudioUnit_AUAudioInputOutputUnit {
 
 		[NullAllowed, Export ("setOutputProvider:")]
 		void SetOutputProvider (AURenderPullInputBlock provider);
-}
+
+		// the following are properties but we cannot have properties in Categories.
+		[Mac (10, 13), NoWatch, NoiOS, NoTV]
+		[Export ("deviceInputLatency")]
+		double GetDeviceInputLatency ();
+
+		[Mac (10, 13), NoWatch, NoiOS, NoTV]
+		[Export ("deviceOutputLatency")]
+		double GetDeviceOutputLatency ();
+
+		[Watch (4, 0), TV (11, 0), Mac (10, 13), iOS (11, 0)]
+		[Export ("running")]
+		bool IsRunning ();
```

Comment:
AUAudioUnit_AUAudioInputOutputUnit is a protocol, AFAIK we cannot have optional properties and we have to have methods. Same thing happens with all the other methods GetDeviceOutputLatency, GetDeviceInputLatency, GetOutputProvider etc...